traffic_ops/app/lib/API/Cachegroup.pm

```diff
@@ -380,7 +380,7 @@ sub by_parameter_id {
 		}
 	}
 
-	$self->success( { cachegroups => \@data } );
+	$self->success( { cachegroups => \@data }, "API path is deprecated use /cachegroupparameters instead");
```

Comment:
"Success" is a bit of a weird type of message for something that's more of a warning about a deprecation. This should use the "warning" level instead, imo.

Reply:
what are the alert "levels"? success, error, warning and info?

Reply:
i thought you could return an array of alerts. if so, what about 1 success message (for the successful assignment of a param to a cg) and 1 warning message with the deprecation message? or is that weird?

Reply:
No I think that is what we want to do! THat would match the work that was done #3954